Author: Ming Hua <minghua-guest@users.alioth.debian.org>
Status: all relevant changes applied in upstream svn since r253
patch can be dropped past 1.4.9
Description: Update to conform Desktop Entry Specification version 1.0.  The "Icon"
key uses a hardcoded path because scim-setup.desktop is generated
without any variable substitution.  Maybe it can use a name without
path or extension instead, needs more investigation.
--- a/extras/setup/scim-setup.desktop.in
+++ b/extras/setup/scim-setup.desktop.in
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=SCIM Input Method Setup
 _Comment=Setup utility for Smart Common Input Method platform
 Exec=scim-setup
-Icon=scim-setup.png
+Icon=scim-setup
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=Applications;Settings;
+Categories=Settings;
